extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java

```diff
@@ -166,12 +172,23 @@ void requireBody(BuildProducer<RequireBodyHandlerBuildItem> requireBodyHandlerPr
         requireBodyHandlerProducer.produce(new RequireBodyHandlerBuildItem());
     }
 
+    @Record(ExecutionTime.RUNTIME_INIT)
+    @BuildStep
+    void buildSchemaEndpoint(
+            BuildProducer<RouteBuildItem> routeProducer,
+            SmallRyeGraphQLRecorder recorder,
+            SmallRyeGraphQLRuntimeConfig runtimeConfig) {
+
+        Handler<RoutingContext> schemaHandler = recorder.schemaHandler(runtimeConfig);
+        routeProducer.produce(
+                new RouteBuildItem(quarkusConfig.rootPath + SCHEMA_PATH, schemaHandler, HandlerType.BLOCKING));
```

Comment:
So we always expose this one?

Reply:
Yes, except that you can also disable in runtime, with the UI.